packages/web-ui/src/ds/atoms/TextArea/index.tsx

```diff
@@ -55,6 +59,8 @@ const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
           placeholder={placeholder}
           className={cn(inputVariants({ size }), className, {
             'border-red-500 focus-visible:ring-red-500': errors,
+            // Account for inner textara padding
+            '!min-h-[calc(100%-theme(spacing.6))]': autoGrow,
```

Comment:
Look how nice is to work with tailwind spacing the have this `theme` function and you pick the right spacing or other token.